src/settings-ui/Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -2125,4 +2125,31 @@ From there, simply click on one of the supported files in the File Explorer and
   <data name="FancyZones_DisableRoundCornersOnWindowSnap.Content" xml:space="preserve">
     <value>Disable round corners when window is snapped</value>
   </data>
-</root>
+  <data name="PowerLauncher_SearchQueryTuningEnabled.Description" xml:space="preserve">
+    <value>Options to fine tune how the results are shown</value>
```

Comment:
`Fine tune how the results are shown` should be enough. Maybe even `Fine tune results ordering`

Reply:
"Fine tune results ordering" works